### PR TITLE
Feature/Added NESTJS and Docker.

### PR DESCRIPTION
**Pull request by ekuritza**
The code of this PR is to add docker and nestJS.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: '3.7'
+
+services: 
+  app:
+    build: .
+    container_name: kuritza
```
Comment (GuilhermeMLS) [inline docker-compose.yml]: Is `container_name` really necessary? If it is, I think `kuritza` is not a good name, as it doesn't tell us anything about what the container actually is.

Also, from the [Docker reference](https://docs.docker.com/compose/compose-file/compose-file-v3/): container names must be unique, you cannot scale a service beyond 1 container if you have specified a custom name. Attempting to do so results in an error.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
+FROM  node
```
Comment (felipemaksy) [inline Dockerfile]: We can specify the version of the node we want to work with, this way we are sure that our application will work as expected.
```
node:14
OR
node:14-alpine
OR
node:16
OR
node:16-alpine
```

We have some variations of the images and each one has a characteristic. Alpine for example is a much smaller and lighter image, you can read more about it [here](https://hub.docker.com/_/node)

**Comment by felipemaksy** [comment]
About commit messages: https://www.conventionalcommits.org/pt-br/v1.0.0/
You don't need to follow 100% of what is in the documentation, you can just use the type and message in the defined pattern, for example:
```
fix: your message here
feat: your message here
refactor: your message here
...
```

**Review by felipemaksy — APPROVED** [review]
Nice!

**Review by GuilhermeMLS — APPROVED** [review]

**Review by ran-j — APPROVED** [review]

**Pull request merged**